src/CBCNodeListViewController.h

```diff
@@ -67,6 +75,9 @@ FOUNDATION_EXTERN CBCNode *_Nonnull CBCCreateNavigationTree(void);
 /** The children of this node. */
 @property(nonatomic, strong, nonnull) NSArray<CBCNode *> *children;
 
+/** The example you wish to debug as the initial view controller. */
```

Comment:
Document that this represents the last view controller to have returned YES to the catalogIsDebug API.